web/packages/teleterm/src/agentCleanupDaemon/agentCleanupDaemon.js

```diff
@@ -59,13 +68,36 @@ if (!process.send) {
   );
 }
 
-const logLine = `parent=${parentPid} agent=${agentPid} self=${process.pid} cluster=${rootClusterUri}`;
+const logger = createLogger({
```

Comment:
I added Winston directly to the cleanup daemon. Capturing logs through the main process wouldn't make sense for the cleanup daemon, as the sole purpose of the cleanup daemon is to take action when the main process exits unexpectedly.